uint8 channel_num           # PCA9685 channel number
float64 target_position     # Taarget position of the servo motor (degrees)
---
bool is_success
string response